src/aap_eda/core/tasking/__init__.py

```diff
@@ -204,14 +207,15 @@ def unique_enqueue(queue_name: str, job_id: str, *args, **kwargs) -> Job:
     Detects if a job with the same id is already enqueued and if it is
     it will return it instead of enqueuing a new one.
     """
-    queue = get_queue(queue_name)
-    job = job_from_queue(queue, job_id)
-    if job:
-        logger.info(
-            f"Skip enqueing job: {job_id} because it is already enqueued"
-        )
-        return job
+    for name in settings.RQ_QUEUES:
+        job = job_from_queue(name, job_id)
+        if job:
+            logger.info(
+                f"Skip enqueing job: {job_id} because it is already enqueued"
+            )
+            return job
     else:
```

Comment:
Should this `else` be indented to match?

Reply:
@hsong-rh that else belongs to the for loop. It is true [it's not necessary](https://docs.astral.sh/ruff/rules/useless-else-on-loop/), but doesn't have an impact either.